### PR TITLE
Today button

### PR DESCRIPTION
**Pull request by stephenrichards**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -14,5 +14,11 @@ gemspec
 # gem 'byebug', group: [:development, :test]
 group :test  do
```
Comment (aliuk2012) [inline Gemfile]: I think this needs to be Test and development because we have removed the dev dependency in the gemspec file and so none of these would be added if bundle install was run on dev mode

File: gov_uk_date_fields.gemspec [hunk 2]
```diff
@@ -16,12 +16,14 @@ Gem::Specification.new do |s|
...
-  s.add_dependency "rails", ">= 4.0"
+  s.add_dependency 'rails', '>= 5.0'
 
-  s.add_development_dependency "pg", "~> 0.18.0"
-  s.add_development_dependency "awesome_print", "~> 1.6"
-  s.add_development_dependency 'mocha', '~> 1.1'
+  # s.add_development_dependency 'pg', '~> 0.18.4'
+  # s.add_development_dependency 'awesome_print', '~> 1.6'
+  # s.add_development_dependency 'mocha', '~> 1.1'
+  # s.add_development_dependency 'rails-controller-testing', '~> 1.0'
```
Comment (aliuk2012) [inline gov_uk_date_fields.gemspec]: Shouldn't we remove all these commented out lines?
